tap-schema/build-all

```diff
@@ -4,9 +4,21 @@
 # values that control the presentation order of schemas.  This is respected by the Portal Aspect.
 #
 ./build mock
-./build idfprod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfint ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfdev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfsso ../yml/dp03.yaml
-./build usdf-dev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
-./build usdf-prod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
+
+./build tap-schema-idfprod-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-idfint-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-idfdev-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-usdf-dev-tap ../yml/dp02_dc2.yaml
+./build tap-schema-usdf-prod-tap ../yml/dp02_dc2.yaml
+
+./build tap-schema-idfprod-obstap ../yml/dp02_obscore.yaml
```

Comment:
In general RSP instances do not need, and in production will not have, separate TAP servers for ObsTAP.  DRP's ObsCore tables must be on the same back-end database and same TAP service as DRP's catalogs, so that JOINs can be done, and for other reasons.

The exception is the (currently still experimental) "live ObsTAP" service, for which we DO need a separate TAP service, with a distinct back-end (namely, the actual Butler Registry Postgres database).  Currently this is only needed for usdf-dev, and hopefully soon for the Summit.